main.py

```diff
@@ -101,6 +104,31 @@ def update_kontaktdaten_interactive(
             input_kontaktdaten_key(
                 kontaktdaten, ["kontakt", "notificationReceiver"], "> Mail: ")
 
+        if configure_notifications:
+            if "pushover_app_token" not in kontaktdaten["kontakt"]:
+                if input("> Benachtigung mit Pushover einrichten? (y/n): ").lower() != "n":
+                    print()
+                    input_kontaktdaten_key(
+                        kontaktdaten, ["kontakt", "pushover_app_token"],
```

Comment:
Bitte diese Sachen nicht innerhalb von `"kontakt"` speichern. Das Datenformat von `"kontakt"` ist durch die impfterminservice API vorgegeben. Der Inhalt wird bei der Terminbuchung an den Server übermittelt.

Lieber `kontaktdaten["pushover_notifications"]` und `kontaktdaten["telegram_notifications"]` verwenden.

Oder alternativ: `kontaktdaten["notifications"]["pushover"]` und `kontaktdaten["notifications"]["telegram"]`.

Im Konkreten Beispiel wäre letzteres:

```suggestion
                        kontaktdaten, ["notifications", "pushover", "app_token"],
```

Reply:
Das hatte ich zuerst so gemacht, habe das dann allerdings angepasst, da die Daten aus "kontakt" in its bereits verfügbar sind.
Um das ganze auszugliedern müsste ein weiteres Dict übergeben werden.
Kann gern mal schauen ob man das elegant lösen kann.
Falls du eine Idee hast wie man das optimal lösen kann, gerne her damit :D

Reply:
Ein `dict` übergeben ist nicht so viel aufwand. Es muss an `@staticmethod terminsuche` und an `__init__` übergeben werden und dann in einem Attribut gespeichert werden. Das hört sich für mich nach 4 Zeilen in `tools/its.py` an. Wenn man die Parameter optional macht (also mit default `= {}`), muss man auch nichts am GUI-Code ändern.

Reply:
Habe ich so umgesetzt und scheint auch zu funktionieren. Kannst du das verifizieren?